conference.js

```diff
@@ -16,6 +16,8 @@ import mediaDeviceHelper from './modules/devices/mediaDeviceHelper';
 
 import {reportError} from './modules/util/helpers';
 
+import Feedback from './modules/UI/Feedback';
```

Comment:
I was thinking about it and I think that the error definition belongs to the Feedback class. I don't want to compare strings also. Do you have a suggestion?